lua/telescope/actions/init.lua

```diff
@@ -57,6 +61,10 @@ function actions.get_current_line()
   return state.get_global_key('current_line')
 end
 
+function actions.get_original_opts()
+  return state.get_global_key('original_opts')
+end
```

Comment:
this seems kind of weird as an action. Not necessarily opposed to it, but it seems a little weird.

Perhaps `_` prefix might make sense here.